Src/VimEditorHost/Implementation/Misc/BasicExperimentationServiceInternal.cs

```diff
@@ -0,0 +1,23 @@
+﻿#if VSVIM_DEV_2019
+using System;
+using System.ComponentModel.Composition;
+using System.ComponentModel.Composition.Primitives;
+using Microsoft.VisualStudio.Language.Intellisense.Utilities;
+using System.Threading;
+using Microsoft.VisualStudio.Text.Utilities;
+
+namespace Vim.EditorHost.Implementation.Misc
+{
+    [Export(typeof(IExperimentationServiceInternal))]
+    internal sealed class BasicExperimentationServiceInternal : IExperimentationServiceInternal

```

Comment:
It's not clear to me why this is needed. Maybe add a comment to a link. Here is the main thing that pops when googling that interface:

* https://github.com/dotnet/roslyn/issues/27428

Reply:
I'll update the comment here. Unfortunately it's not going to be much more informative than the data in the issue you linked to. We're implementing it here because the editor expects it to be implemented. There is no real documentation on what it does. It's unfortunate they didn't use an optional import as the implementation isn't really needed here.